Eggstravaganza/Assets/Scripts/GameNetwork.cs

```diff
@@ -95,7 +101,7 @@ void Update()
 
     void DecrementLobbyTime()
     {
-        if (IsOwner)
+        if (IsServer)
```

Comment:
my bad :grimacing: thanks for updating all this!

Reply:
All good!